packages/edit-post/src/hooks/use-navigate-to-entity-record.js

```diff
@@ -52,7 +55,10 @@ export default function useNavigateToEntityRecord(
 	const { setRenderingMode } = useDispatch( editorStore );
 
 	const onNavigateToEntityRecord = useCallback(
-		( params ) => {
+		async ( params ) => {
+			await registry
+				.resolveSelect( coreStore )
+				.getPostType( params.postType );
```

Comment:
We probably need to `catch` any potential errors here so they don't leak out as this function was previously _not_ async.